docs/user/aws/images/install_upi.svg

```diff
@@ -0,0 +1,370 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
+<!DOCTYPE svg PUBLIC "-//W3C//DTD SVG 1.0//EN" "http://www.w3.org/TR/2001/PR-SVG-20010719/DTD/svg10.dtd">
+<svg width="91cm" height="44cm" viewBox="-302 -191 1813 861" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink">
+  <g>
+    <rect style="fill: #ffffff" x="-300.166" y="-190" width="1810" height="859"/>
+    <rect style="fill: none; fill-opacity:0; stroke-width: 2; stroke: #000000" x="-300.166" y="-190" width="1810" height="859"/>
+  </g>
+  <image x="-282.166" y="-179" width="89" height="89" xlink:href="file:///home/trking/src/openshift/installer/docs/user/aws/images/./AWS-Architecture-Icons_SVG/Light-BG/_General%20AWS/AWS-General_AWS-Cloud_light-bg.svg"/>
```

Comment:
/hold

Need to fix these busted references.

Reply:
> Need to fix these busted references.

Fixed with 7b86189e9 -> f5ee4a584, CC @cuppett, who was curious about how that might work ;).

Reply:
/hold cancel